resources/lang/ep_gui.zh-TW.rc

```diff
@@ -5,430 +5,430 @@ LANGUAGE LANG_CHINESE, SUBLANG_CHINESE_TRADITIONAL
-    IDS_ADV_PAINTDESKTOPVERSION "桌面顯示 Windows 版本資訊"
-    IDS_ADV_CLASSICTHEME        "啟用進階紓解措施來正確繪製和使用經典主題"
-    IDS_ADV_SYSLISTVIEW32       "啟用舊版檢視控制項 SysListView32"
-    IDS_ADV_NOPROPERTIES        "不修改工作列右鍵選單，隱藏「屬性」本視窗的打開連結"
-    IDS_ADV_SYMBOLS             "啟用符號檔案下載"
-    IDS_ADV_PINNEDITEMS         "固定項充當快速啟動（不與活動應用分組）"
-    IDS_ADV_REMOVEEXTRAGAP      "工作列顯示按鈕標籤時，移除固定項周圍的額外間隙"
-    IDS_ADV_DELAY               "登入後的延遲"
-    IDS_ADV_DELAY_0             "無（預設）"
-    IDS_ADV_DELAY_300           "300 ms"
-    IDS_ADV_DELAY_600           "600 ms"
-    IDS_ADV_DELAY_1000          "1 s"
-    IDS_ADV_DELAY_1500          "1.5 s"
-    IDS_ADV_DELAY_2000          "2 s"
-    IDS_ADV_DELAY_3000          "3 s"
-    IDS_ADV_DELAY_4000          "4 s"
-    IDS_ADV_DELAY_5000          "5 s"
-    IDS_ADV_DELAY_6000          "6 s"
-    IDS_ADV_DELAY_8000          "8 s"
-    IDS_ADV_DELAY_10000         "10 s"
-    IDS_ADV_XAMLSOUNDS          "在檔案總管的 XAML 檢視中啟用 UI 聲音"
+IDS_ADV_DISCLAIMER          "請仔細閱讀檔案後再修改這些設定。"
+IDS_ADV_LEARN               "檔案"
+IDS_ADV_CONSOLE             "啟用控制台"
+IDS_ADV_MEMCHECK            "轉儲記憶體洩漏紀錄"
+IDS_ADV_AUTOHIDE            "按兩下工作列以切換自動隱藏 (僅在工作列鎖定時有效)"
+IDS_ADV_PAINTDESKTOPVERSION "桌面顯示 Windows 版本浮水印"
+IDS_ADV_CLASSICTHEME        "啟用進階緩解措施以正確繪製和使用經典主題"
+IDS_ADV_SYSLISTVIEW32       "啟用舊版檢視控制項 SysListView32"
+IDS_ADV_NOPROPERTIES        "不修改工作列右鍵選單，隱藏 ExplorerPatcher 設定入口（內容）"
+IDS_ADV_SYMBOLS             "啟用符號檔案下載"
+IDS_ADV_PINNEDITEMS         "以固定項目作為快速啟動 (不把固定項目和活動中的應用程式分組)"
+IDS_ADV_REMOVEEXTRAGAP      "工作列顯示按鈕標籤時，移除固定項目周圍的額外間隙"
+IDS_ADV_DELAY               "登入後的延遲"
+IDS_ADV_DELAY_0             "無(預設)"
+IDS_ADV_DELAY_300           "300 ms"
+IDS_ADV_DELAY_600           "600 ms"
+IDS_ADV_DELAY_1000          "1 秒"
+IDS_ADV_DELAY_1500          "1.5 秒"
+IDS_ADV_DELAY_2000          "2 秒"
+IDS_ADV_DELAY_3000          "3 秒"
+IDS_ADV_DELAY_4000          "4 秒"
+IDS_ADV_DELAY_5000          "5 秒"
+IDS_ADV_DELAY_6000          "6 秒"
+IDS_ADV_DELAY_8000          "8 秒"
+IDS_ADV_DELAY_10000         "10 秒"
+IDS_ADV_XAMLSOUNDS          "在檔案總管的 XAML 檢視中啟用 UI 音效"
 
-    IDS_ABOUT                   "關於"
+IDS_ABOUT                   "關於"
 
-    IDS_ABOUT_VERSION           "版本 %d.%d.%d.%d%s"
-    IDS_ABOUT_PROJECT           "該項目立志增強 Windows 工作體驗。"
-    IDS_ABOUT_AUTHOR            "Valentin-Gabriel Radu 榮譽出品。"
-    IDS_ABOUT_OS                "系統版本 %s OS Build %d.%d。"
-    IDS_ABOUT_GITHUB            "查看項目 GitHub"
-    IDS_ABOUT_WEBSITE           "瀏覽官網"
-    IDS_ABOUT_DONATE            "捐贈支援"
-    IDS_ABOUT_FAQ               "常見問題"
-    IDS_ABOUT_SETTINGS          "設定指南"
-    IDS_ABOUT_IMPORT            "匯入設定"
-    IDS_ABOUT_EXPORT            "匯出設定"
-    IDS_ABOUT_EXPORT_SUCCESS    "設定匯出完成。"
-    IDS_ABOUT_RESET             "復原預設設定"
+IDS_ABOUT_VERSION           "版本 %d.%d.%d.%d%s"
+IDS_ABOUT_PROJECT           "此專案讓 Windows 的工作環境更加高效率。"
+IDS_ABOUT_AUTHOR            "Valentin-Gabriel Radu 榮譽出品。"
+IDS_ABOUT_OS                "系統版本 %s 組建 %d.%d"

```

Comment:
Keep the dot at the end.

Reply:
Thank you for review. I have corrected the above issues.